.github/release-please.yml

```diff
@@ -16,6 +16,9 @@ handleGHRelease: true
 packageName: cloud-sql-proxy
 releaseType: simple
 versionFile: 'cmd/version.txt'
+changelogPath: 'CHANGELOG.md'
```

Comment:
This and the one below are probably unnecessary. Perhaps just adding releaseType will do the trick as it has no default.

Reply:
ok, we'll try without the explicit setting.